CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,5 @@
+# Code of Conduct
+
+One of Cesium's strengths is our community.  Our contributors and users are pushing the 3D geospatial field to amazing new levels.  We rely on an open, friendly, inclusive environment to facilitate this.  As such, we follow the [Contributor Covenant](http://contributor-covenant.org/)'s [Code of Conduct](http://contributor-covenant.org/version/1/4/code-of-conduct.md) to ensure a harassment-free experience in the Cesium community.  Any unacceptable behavior can be confidentially sent to the core team at pcozzi@agi.com.
```

Comment:
Why `pcozzi@agi.com`  and not the @ Cesium email? I think it should be addressed to the dev-rel team, via `hello@cesium.com`, or if we're to encourage sending to a specific person we should mention their name (like send it to Patrick at <email>) to make it friendlier/more approachable. It's weird that this says it's going to the "core team" but seems to be going to one person.

Reply:
This is a direct copy of Cesium's Code of Conduct, any changes should probably be made there first and filtered to other repositories.

Reply:
Just to be clear, I'm fine with changing it, but please open a Cesium PR with the exact wording you want so that it gets changed there and then I'll copy it to hear as well.